src/components/pager/js/Pager.js

```diff
@@ -295,8 +295,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     });
 
     fluid.pager.previousNext.update = function (that, disabledStyle, newModel) {
-        that.previous.toggleClass(disabledStyle, newModel.pageIndex === 0);
-        that.next.toggleClass(disabledStyle, newModel.pageIndex === newModel.pageCount - 1);
+        var isFirstPage = (newModel.pageIndex === 0);
```

Comment:
Actually, a better solution is to use `modelRelay` to calculate `isFirstPage` & `isLastPage` based on the source model values `{pager}.model.pageIndex` and `{pager}.model.pageCount`, then attach model listeners that listen to the model changes of `isFirstPage` & `isLastPage` to update disabled states.

An example:
Model Relay: https://github.com/GPII/first-discovery/blob/master/src/js/panels.js#L640-L646
The corresponding model listener: https://github.com/GPII/first-discovery/blob/master/src/js/panels.js#L671-L674